README.md

```diff
@@ -168,4 +209,19 @@ module "ubuntu_desktop" {
 | <a name="output_public_subnet_cidr_block"></a> [public\_subnet\_cidr\_block](#output\_public\_subnet\_cidr\_block) | The public subnet CIDR block |
 | <a name="output_public_subnet_id"></a> [public\_subnet\_id](#output\_public\_subnet\_id) | The public subnet ID |
 | <a name="output_vpc_arn"></a> [vpc\_arn](#output\_vpc\_arn) | The arn of the VPC |
-| <a name="output_vpc_id"></a> [vpc\_id](#output\_vpc\_id) | The ID of the VPC |
\ No newline at end of file
+| <a name="output_vpc_id"></a> [vpc\_id](#output\_vpc\_id) | The ID of the VPC |
+
+## Contributing
+
+### Bug Reports & Feature Requests
+
+Please use the [issue tracker](https://github.com/InterweaveCloud/terraform_aws_ubuntu_desktop_vm/issues) to report any bugs or file feature requests.
+
+## Contributors
+| Name | Role |
+|------|------|
+| [Faizan Raza](https://www.linkedin.com/in/faizan-raza-997808206/) | Lead Developer |
+| [Muhammad Hasan](https://www.linkedin.com/in/muhammad-hasan-b2553221a/) | Developer |
```

Comment:
you are the lead since you wrote it mainly

Reply:
done